### ISSUE TITLE
Short text output

### ISSUE THREAD

**Issue by George3d6** [mindsdb/lightwood#218]
We now support unordered/bag-of-words style output from lightwood, however it might be useful to implement something like this fort ordered yet very short text as well.

Basically, allowing columns encoded via the short-text encoder (ShortText type) to be targets. The mixer would just be predicting ordered one-hot encoded representations in this case. With an all 0s vector indicating a stopping point.

Given that we have something like:

Cat = [0 0 1]
Dog = [0 1 0]
The = [1 0 0]

The mixer predicting `[1 0 0 0 1 0 0 0 0]` would be == `The Dog`
The mixer predicting `[0 0 1 0 0 0 0 1 0]` would be == `Cat`
The mixer predicting `[1 0 0 0 0 1 0 1 0]` would be == `The Cat Dog`

The current sort text encoder already supports encoding/decoding these representations in `concat` mode, we just need to make sure `concat` mode is always selected when `target == True` in the encoder's args.

The other problem here is finding a dataset to test this on and finding an appropriate loss function.

I think @btseytlin you had some ideas as to the ideal loss to be used here and maybe some of the datasets you provided could work as well, if so, please do tell.

**Comment by maximlopin** [comment]
my thoughts on loss function:
If the target is `The cat dog -> [1, 0, 0, 0, 1, 0, 0, 0, 1]`, and the prediction is `The Dog Cat -> [1, 0, 0, 0, 0, 1, 0, 1, 0]`,
the loss would be `CrossEntropy([1, 0, 0], [1, 0, 0]) + CrossEntropy([0, 1, 0], [0, 0, 1]) + CrossEntropy([0, 0, 1], [0, 1, 0])`.
Loss function should ignore vectors following the "stopping point" vector.

**Comment by maximlopin** [comment]
partially resolved by https://github.com/mindsdb/lightwood/pull/219
now need to find a good dataset
